README.md

```diff
@@ -96,39 +99,42 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** What is happening on the first couple of lines of the new file you've created?
 
-* **Your Answer:** 
-
+* **Your Answer:**
+We are designating the url at which our app will be hosted. Which will be localhost 5000 while we are developing and if we were to productionize this, we would add the final url.
 ---
 
-- [ ] Let's store the token in LocalStorage with a key of `journal-app`.
+- [x] Let's store the token in LocalStorage with a key of `journal-app`.
 
 * **Question:** Why are we storing the token?
 
 * **Your Answer:**
-
+So we stay logged in
```

Comment:
Yes, it would be frustrating for the user to have to log in every time they refreshed the page or navigated to a route manually.